plugins/out_stackdriver/stackdriver.c

```diff
@@ -936,6 +937,32 @@ static int stackdriver_format(struct flb_config *config,
     /* Count number of records */
     array_size = flb_mp_count(data, bytes);
 
+    /* 
+     * Search each entry and validate insertId.
+     * Reject the entry if insertId is invalid.
+     * If all the entries are rejected, stop formatting.
+     * 
+     */
+    off = 0;
+    msgpack_unpacked_init(&search_insert_id);
```

Comment:
rename the unpacker, in general we use the variable name ```result``` on all unpacker calls.

Reply:
you can reuse it if needed later.

Reply:
Done